cli/compose/convert/service.go

```diff
@@ -158,6 +158,7 @@ func Service(
 			Placement: &swarm.Placement{
 				Constraints: service.Deploy.Placement.Constraints,
 				Preferences: getPlacementPreference(service.Deploy.Placement.Preferences),
+				MaxReplicas: service.Deploy.Placement.MaxReplicas,
```

Comment:
This and a bunch of other changes to `cli/compose/*` seem like they are from some other work?

Reply:
Sorry I need to rebase my branch